### ISSUE TITLE
\mathrm and other commands are missing in the completion items of the current master

### ISSUE THREAD

**Issue by tamuratak** [James-Yu/LaTeX-Workshop#2047]
`\mathrm` and other commands are missing in the completion items of the current master. `data/packages/latex-document_cmd.json` is not loaded. 

![スクリーンショット 2020-04-22 15 20 41](https://user-images.githubusercontent.com/10665499/79948149-44b35200-84ae-11ea-9d27-a702bb71ffdc.png)

**Comment by jlelong** [comment]
You are right. It seems that `data/packages/latex-document_cmd.json` has never been loaded, which created this issue when I sanitized the file `data/commands.json`. Maybe I was a bit too agressive.

**Comment by jlelong** [comment]
Actually, it turns out that 3 major completion files are never loaded:
```
latex-dev_cmd.json
latex-document_cmd.json
latex-document_env.json
```

It is clear that the last two must be loaded by default but I am not so sure about `latex-dev_cmd.json`

**Comment by tamuratak** [comment]
I was thinking that it was intentional not loading `latex-document_cmd.json`, which enables us to control snippets of basic LaTeX commands by hand. I do not know much about snippets of LaTeX Workshop.

There are two things to say.

Now, we see duplicates of some commands.

![スクリーンショット 2020-04-22 19 59 44](https://user-images.githubusercontent.com/10665499/79974226-e305de80-84d3-11ea-8b77-d92f6cd42393.png)

I prefer less used commands, `\left\backslash` and others, to be removed since they push out `\left\{` from the suggestion.

![スクリーンショット 2020-04-22 20 01 23](https://user-images.githubusercontent.com/10665499/79974382-37a95980-84d4-11ea-9616-805dbfab0361.png)

**Comment by jlelong** [comment]
Good point! 
So it may be wiser not to load `latex-document_{cmd,env}.json` but to only add some of their entries into `commands.json` and `environments.json`.

**Comment by jlelong** [comment]
>  I do not know much about snippets of LaTeX Workshop.

There are two different systems working in parallel

1. The standard snippet mechanism of VSCode based on the content of `snippets/latex.json`. Completion based on this file is triggered after two characters are typed when `editor.suggest.showSnippets` is true.

1. The internal LaTeX-Workshop completion provider triggered by `\`. The data used to populated completion come from the directory `data`. The following files are always loaded

    - `bibtex-entries.json`: completion in bibtex files
    - `classnames.json`: list of LaTeX classes, used for completion in `\documentclass`
    - `packagenames.json`: list of LaTeX packages, used for completion inside `\usepackage`
    - `commands.json`: a set of default LaTeX commands
    - `environments.json`: a set of default LaTeX environments

    On top of that, we include some additional commands and environments  based on the packages that are used.
